### ISSUE TITLE
Error when using most vpp filters on Coffeelake

### ISSUE THREAD

**Issue by otuncelli** [rigaya/QSVEnc#179]
I can only use `auto/simple/advanced` options for `vpp-resize` without an error. Most vpp filters are also giving me the same error.

I don't know if this helps but I'm using my Nvidia 1050 ti as my main GPU.

Command line:
`QSVEncC64.exe --avsync forcecfr --icq 18 --profile main10 --quality best --output-depth 10 --codec hevc --vpp-resize-mode quality --metadata copy --audio-copy --input input.mp4 --output output.mp4 --output-res -2x1080 --vpp-resize lanczos2`

is giving me this output:
```
--------------------------------------------------------------------------------
output.mp4
--------------------------------------------------------------------------------
cop.SingleSeiNalUnit value changed off -> auto by driver
QSVEncC (x64) 7.50 (r3150) by rigaya, Oct 18 2023 12:41:11 (VC 1935/Win)
OS             Windows 10 x64 (17763) [CP1252]
CPU Info       Intel Core i5-8500 @ 3.00GHz [TB: 3.88GHz] (6C/6T) <Coffeelake>
GPU Info       Intel UHD Graphics 630 (24EU) 350-1100MHz [65W] (26.20.100.7642)
Media SDK      QuickSyncVideo (hardware encoder) PG, 2nd GPU, API v1.30
Async Depth    3 frames
Buffer Memory  d3d11, 27 work buffer
Input Info     avqsv: H.264/AVC, 3840x2160, 24/1 fps
VPP            cspconv(nv12 -> yv12(16bit))
               resize(lanczos2): 3840x2160 -> 1920x1080
               cspconv(yv12(16bit) -> p010)
AVSync         forcecfr
Output         H.265/HEVC(yuv420 10bit) main10 @ Level 5 (high tier)
               1920x1080p 1:1 24.000fps (24/1fps)
               avwriter: hevc, aac => mp4
Target usage   1 - best
Encode Mode    ICQ (Intelligent Const. Quality)
ICQ Quality    18
QP Limit       min: none, max: none
Ref frames     6 frames
Bframes        3 frames, B-pyramid: off
Max GOP Length 240 frames
Ext. Features  WeightP WeightB GPB ctu:32 sao:none
OpenCL platform not associated with d3d11 device.
OPENCL: Failed to get OpenCL interop [in].
Break in task OPENCL: null pointer..
avout: File header not written, unexpected error!
**Last line repeats multiple times.**
...
...
...
encoded 0 frames, 0.00 fps, 0.00 kbps, 0.00 MB
encode time 0:00:00, CPULoad: 12.3%


QSVEncC.exe finished with error!
```

**Comment by otuncelli** [comment]
Alright, driver update fixed the issue.

**Comment by Discograp** [comment]
Hello! 
Please answer, this bug was in which version and did you update the driver to which version?

**Comment by otuncelli** [comment]
> Hello! Please answer, this bug was in which version and did you update the driver to which version?

I was using the version `26.20.100.7642` which can be seen in the console output above. Now, I'm using the version `31.0.101.2115`

I had updated it before posting the issue here but Windows Update seems to have reverted it back to the old driver some reason. I did some research on this and found out this might happen.
https://www.intel.com/content/www/us/en/support/articles/000087834/graphics.html

It still happens sometimes and I have to manually switch to the latest driver using Device Manager.
